src/stringlist-to-array.js

```diff
@@ -0,0 +1,9 @@
+'use strict'
+
+// Converts a go-ipfs "stringList" to an array
+// { Strings: ['A', 'B'] } --> ['A', 'B']
+function stringlistToArray (res, cb) {
+  cb(null, res.Strings || [])
+}
+
+module.exports = stringlistToArray
```

Comment:
We shouldn't change this on go-ipfs side for this PR. If we do that, it's an API breaking change that we need to handle with proper care. That's why I put it in one file, so that it's wrapped and isolated and if/once we change the return values, we only need to change change it in this file and whichever command is using the current API doesn't need changes. Obviously once you do that, we don't need the stringlist-to-array.js conversion function and can get rid of the file completely, but I recommend we keep it as such now so that we isolate the data conversion to one place. Does that make sense?